snuba/datasets/errors_replacer.py

```diff
@@ -413,14 +413,18 @@ def get_timestamp_condition(msg_field: str, operator: str) -> str:
             ]
         )
     else:
-        event_id_lhs = "cityHash64(event_id)"
-        event_id_list = ", ".join(
-            "cityHash64(toUUID('%s'))" % uuid.UUID(eid) for eid in event_ids
+        event_id_lhs = "event_id"
+        event_id_list = ", ".join("'%s'" % uuid.UUID(eid) for eid in event_ids)
+
+    if old_primary_hash:
+        old_primary_hash_condition = (
+            " AND (%(old_primary_hash)s IS NULL OR primary_hash = %(old_primary_hash)s)"
```

Comment:
Do we need the first condition anymore? Seems like it won't be null anymore in here since we have that check on line 419 now